flink-table/flink-table-api-scala/src/main/scala/org/apache/flink/table/api/expressionDsl.scala

```diff
@@ -1160,6 +359,13 @@ trait ImplicitExpressionConversions {
     }
   }
 
+  implicit class FieldExpression(val sc: StringContext) {
+    def $(args: Any*): Expression = unresolvedRef(sc.s(args: _*))
+  }
+
+  implicit def apiExpressionToExpression(expr: ApiExpression): Expression =
```

Comment:
Is this really necessary or just convenience? A ScalaDoc would help.

Reply:
It's not necessary. Let's remove it.